File: happykopiApp/src/app/core/dtos/order/product-configuration-result.dto.ts
import { AllAvailableAddonDto } from './all-available-addon.dto';
import { OrderVariantAddOnDto } from './order-variant-addon.dto';
import { OrderVariantIngredientDto } from './order-variant-ingredient.dto';
import { OrderVariantDto } from './order-variant.dto';

export interface ProductConfigurationResultDto {
  variants: OrderVariantDto[];
  ingredients: OrderVariantIngredientDto[];
  addOns: OrderVariantAddOnDto[];
  allAvailableAddons: AllAvailableAddonDto[];

}

File: happykopiAPI/happykopiAPI/DTOs/Order/Outgoing Data/ProductConfigurationResultDto.cs
namespace happykopiAPI.DTOs.Order.Outgoing_Data
{
    public class ProductConfigurationResultDto
    {
        public List<OrderVariantDto> Variants { get; set; } = new();
        public List<OrderVariantIngredientDto> Ingredients { get; set; } = new();
        public List<OrderVarianAddontDto> AddOns { get; set; } = new();
        public List<OrderModifierSummaryDto> AllAvailableAddons { get; set; } = new();

    }
}